<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Draw</class>
 <widget class="QMainWindow" name="Draw">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>211</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>Заливка фона</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>синий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>красный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>зелёный</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_2">
       <item>
        <property name="text">
         <string>Цвет линий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>чёрный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>красный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>синий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>зелёный</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_3">
       <item>
        <property name="text">
         <string>Тип линий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>solid</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>dash</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>dot</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Толщина линий</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Перерисовать</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
